src/Chapter1.hs

```diff
@@ -542,7 +543,15 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z
+    | y <= x && x <= z = x
+    | z <= x && x <= y = x
+    | x <= y && y <= z = y
+    | z <= y && y <= x = y
+    | x <= z && z <= y = z
+    | otherwise = z
```

Comment:
You can save a few lines of code, and put these last two cases under a single `otherwise` :slightly_smiling_face: